hermes/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: format lint test test-unit govet govulncheck
+
+## format: Install and run goimports and gofumpt
```

Comment:
I believe those comments are redundant, as its content is self-explanatory.

Reply:
The comments are used for the `make help` command, I forgot by mistake

Reply:
but I removed the makefile